python_bindings/correctness/addconstant_generator.cpp

```diff
@@ -0,0 +1,65 @@
+#include "Halide.h"
+
+using namespace Halide;
+
+class AddConstantGenerator : public Halide::Generator<AddConstantGenerator> {
+public:
+    Input<uint8_t> constant_uint8{"constant_uint8"};
```

Comment:
Added `Input<bool>`. Note that `Input<Buffer<bool>>` isn't expected to work, since the Python buffer API doesn't (it seems) have any support for bit arrays, so there's no good way to create such a buffer, in Python. But I added a test for the generator itself, and for the fact that calling the generated function will throw an exception.